src/platforms/dotnet/profiling/index.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Set Up Profiling
+description: "Learn how to enable profiling in your app if it is not already set up."
+sidebar_order: 5000
+---
+
+<PlatformContent includePath="profiling/index/preface" />
+
+With [profiling](/product/profiling/), Sentry allows you to collect and analyze performance profiles from real user devices in production to give you a complete picture of how your application performs in a variety of environments.
+
+<Note>
+
+.NET profiling alpha is available .NET 6.0+ (tested on .NET 7.0 & .NET 8.0 as well) for:
+- Windows
+- Linux
+- macOS
+- iOS
+- macCatalyst
+
+</Note>
+
+## Enable Profiling
+
+To enable profiling, set the `ProfilesSampleRate`. 
+Additionally, for all platforms except iOS/Mac Catalyst, you need to add a dependency on the `Sentry.Profiling` NuGet package.
```

Comment:
```suggestion
Additionally, for all platforms except iOS/Mac Catalyst, you need to add a dependency on the `Sentry.Profiling` NuGet package.

```shell {tabTitle:.NET CLI}
dotnet add package Sentry.Profiling -v {{@inject packages.version('sentry.dotnet', '4.0.0-beta.8') }}
```
```

Reply:
P. S. you need to use four backticks in order to use 3 in the actual suggestion

````suggestion
Additionally, for all platforms except iOS/Mac Catalyst, you need to add a dependency on the `Sentry.Profiling` NuGet package.
	
	```shell {tabTitle:.NET CLI}
	dotnet add package Sentry.Profiling -v {{@inject packages.version('sentry.dotnet', '4.0.0-beta.8') }}
	```
````